addon/ShareProvider.js

```diff
@@ -403,7 +404,14 @@ ShareProvider.prototype = {
         if (!node || node.id !== id) {
           return;
         }
-        node.setAttribute("image", data.url("content/img/glyph-share-16.svg"));
```

Comment:
actually, I just realized that icon is still used in the context menu. I'll leave that for another time.